docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -42,7 +40,7 @@ Note how changing the value of the start date section will call `onChange` even
 
 {{"demo": "LifeCycleDateRangeField.js", "defaultCodeOpen": false}}
 
-## Pickers lifecycle
+## Lifecycle on pickers: `onClose`
```

Comment:
WDYT about avoiding the usage of code in headings?
They do not seem to be transferred into the side menu and also break spacing. 🙈 
<img width="402" alt="Screenshot 2024-07-17 at 13 11 11" src="https://github.com/user-attachments/assets/db13aa4d-b047-4f9e-bad5-c8b8b4647701">

```suggestion
## Lifecycle on pickers: "onClose"
```

The same would apply to the h3 sections. 🤔 

Or we need to ditch the `display: flex` on the `anchor` element. 🙈

Reply:
I'm fine removing the code :+1: